superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx

```diff
@@ -71,7 +71,7 @@ export default function CccsGrid({
   filters: initialFilters = {},
 }: CccsGridProps) {
 
-  const [filters, setFilters] = useState(initialFilters);
+  const [, setFilters] = useState(initialFilters);
```

Comment:
Just double checking. Filters seems important. Do we really want it gone? :D

Reply:
Yep, infact we need it gone!